### PR TITLE
Adding SNOW updates blog post

### PR DESCRIPTION
**Pull request by IsaacCalligeros95**
# About

A number of changes to ITSM Providers, mostly ServiceNow. These changes are QOL improvements with some new functionality added.

# Release

2024.2

# Blog image idea
I've just used the existing image from the original SNOW blog post.

# Before you submit your post

Check everything off this list before you submit your post for review. 

- [x] I'm familiar with the [Octopus voice and style](https://www.octopus.design/932c0f1a9/p/13df3a-voice-and-style), the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content), and [writing educational content](https://www.octopus.design/932c0f1a9/p/137b06-educational-content---how-we-write-about-what-we-do).
- [x] I've checked the content using Grammarly (Octopus has a corporate license) and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [x] I've added alt text for all the images I've included (alt text describes an image to people unable to see it, 125 characters max)
- [x] The draft is complete and this post is ready to be reviewed. (Please mark the post 'ready for review' at the bottom of the PR.)
- [ ] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)
- [ ] I'll draft social media posts for Twitter and LinkedIn. I'll post them for review in #topic-social-media after the post has been edited or after the PR has been merged.

## How to review this PR

If there's anything you'd like reviewers to focus on, mention it here.

## Publication date
Almost all changes relating to this post have been merged. It should be fine to publish whenever is suitable.

### CODE REVIEW

**Review by benPearce1 — APPROVED** [review]

File: blog/2024-04/servicenow-updates/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+isFeatured: false
+tags:
+- Product
+- DevOps
+- Integrations
+---
+
+Octopus added support for IT Service Management (ITSM), integrating with ServiceNow and Jira Service Management in 2022.3. Since then, we've received feedback highlighting points of friction and areas for improvement. We've revisited this feature, adding new functionality and some quality-of-life changes.
+
+## Tasklog
```
Comment (benPearce1) [inline blog/2024-04/servicenow-updates/index.md:19]:
```suggestion
## Task log
```

File: blog/2024-04/servicenow-updates/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Previously ITSM precondition checks would only show details on the Task Summary tab. We've now added logging to the process that creates the change request (CR). If the creation is successful this surfaces the CR number and a link to it. If unsuccessful, errors relating to the CR creation are now logged. These logging changes enhance visibility and troubleshooting during CR creation, particularly around permission and configuration issues.
+
+![ITSM tasklogs](itsm-tasklog.png "width=500")
+
+## Emergency changes
+
+Octopus now supports creating emergency changes. To create an Emergency change request, you can now Select the `Emergency Change` setting on the deployment creation page. Emergency changes are intended for scenarios such as resolving a major incident or implementing a security patch.
+
+![ITSM tasklogs](itsm-deployment-settings.png "width=500")
```
Comment (hnrkndrssn) [inline blog/2024-04/servicenow-updates/index.md:29]:
```suggestion
![ITSM deployment settings](itsm-deployment-settings.png "width=500")
```

File: blog/2024-04/servicenow-updates/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Tasklog
+
+Previously ITSM precondition checks would only show details on the Task Summary tab. We've now added logging to the process that creates the change request (CR). If the creation is successful this surfaces the CR number and a link to it. If unsuccessful, errors relating to the CR creation are now logged. These logging changes enhance visibility and troubleshooting during CR creation, particularly around permission and configuration issues.
+
+![ITSM tasklogs](itsm-tasklog.png "width=500")
+
+## Emergency changes
+
+Octopus now supports creating emergency changes. To create an Emergency change request, you can now Select the `Emergency Change` setting on the deployment creation page. Emergency changes are intended for scenarios such as resolving a major incident or implementing a security patch.
```
Comment (benPearce1) [inline blog/2024-04/servicenow-updates/index.md:27]:
```suggestion
Octopus now supports creating emergency changes. To create an Emergency change request, you can now select the `Emergency Change` setting on the deployment creation page. Emergency changes are intended for scenarios such as resolving a major incident or implementing a security patch.
```

File: blog/2024-04/servicenow-updates/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+![ITSM tasklogs](itsm-deployment-settings.png "width=500")
+
+
+## Transition to closed
```
Comment (hnrkndrssn) [inline blog/2024-04/servicenow-updates/index.md:32]: Should we include a screenshot of this new setting? Not sure it's needed, I'll leave it up to you.

Comment (IsaacCalligeros95) [reply blog/2024-04/servicenow-updates/index.md:32]: I felt with the other screenshots, it wasn't that relevant and would just crowd the post. It's included in the docs.

File: blog/2024-04/servicenow-updates/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+For example to set the `Assigned To` or `Short Description` fields you can use the following:
+
+| Field | Variable | Example Value|
+|--|--|--|
+|Assigned To|Octopus.ServiceNow.Field[assigned_to]|beth.anglin|
+|Short Description|Octopus.ServiceNow.Field[short_description]Custom Short Description with #{SomeVariable} #{Octopus.Deployment.Id}|
+
+:::hint
+Setting a `Short Description` will over-ride the auto generated Octopus description. [Title text matching](#title-text-matching) means this will automatically progress the deployment unless the resolved description is unique. This can be done by including variables like the deployment or environment Id.
```
Comment (hnrkndrssn) [inline blog/2024-04/servicenow-updates/index.md:48]: The title text matching sentence seems to be missing a word or two as it reads a bit strange... 🤔

Comment (hnrkndrssn) [reply blog/2024-04/servicenow-updates/index.md:48]: And that link won't go anywhere as there's no `#title-text-matching` anchor in this post.

Comment (IsaacCalligeros95) [reply blog/2024-04/servicenow-updates/index.md]: Missed that, that section was copied across from the docs.

**Review by hnrkndrssn — APPROVED** [review]

**Pull request merged**